src/gateway/wechat-gateway.ts

```diff
@@ -335,7 +335,26 @@ export class WechatGateway extends EventEmitter {
         const rawData: any = []
         let dataLen = 0
 
-        if (response.statusCode !== 200) {
+        if (response.statusCode === 302) {
+          const location = response.headers.location || res.commandUrl
+          log.info(PRE, '302 location', location)
+          let hostname: string | undefined
+          if (/^(http|https):\/\//.test(location)) {
+            const hostnameReg = new RegExp(/(?<=\/\/).*?(?=\/)/)
+            const hostnameBody = location.match(hostnameReg)
+            hostname = hostnameBody ? hostnameBody[0] : location
+            log.info(PRE, 'hostname', hostname)
```

Comment:
This is not necessary.

Reply:
Add this log to check the REG is right or not, maybe I should use log.debug() instead log.info()?

Reply:
Yep, you might need `log.silly()`

Reply:
`log.info()` should only log out informations that our user needed. And for this case, we log out the `location` for temporary investigation purpose, after we solve this problem entirely, we should remove the info log of location.